python/tests/image/test_imageIO.py

```diff
@@ -23,9 +23,8 @@
 from pyspark.sql.functions import col, udf
 from pyspark.sql.types import BinaryType, StringType, StructField, StructType
 
-from pyspark.ml.image import ImageSchema
-
 from sparkdl.image import imageIO
+from pyspark.ml.image import ImageSchema
```

Comment:
group with pyspark imports above (i hope i didn't ask you to move it down here in the last review...)